cmds/webboot/webboot_test.go

```diff
@@ -104,15 +105,46 @@ func TestDownloadOption(t *testing.T) {
 
 func TestISOOption(t *testing.T) {
 	iso := &ISO{
-		label: "TinyCorePure64-10.1.iso",
-		path:  "./testdata/Tinycore/TinyCorePure64-10.1.iso",
+		label: "TinyCorePure64.iso",
+		path:  "./testdata/dirlevel1/dirlevel2/TinyCorePure64.iso",
 	}
 
 	if err := iso.exec(); err != nil {
-		t.Errorf("Fail to execute, error msg: %+v", err)
+		t.Errorf("Fail to execute iso.exec(): %+v", err)
 	}
 
 	if _, err := os.Stat(iso.path); err != nil {
-		t.Errorf("Fail to find the iso file, error msg: %+v", err)
+		t.Errorf("Fail to find the iso file: %+v", err)
+	}
+}
+
+func TestDirOption(t *testing.T) {
+	wanted := &ISO{
+		label: "TinyCorePure64.iso",
+		path:  "testdata/dirlevel1/dirlevel2/TinyCorePure64.iso",
+	}
+
+	uiEvents := make(chan ui.Event)
+	input := []string{"0", "<Enter>", "0", "<Enter>", "0", "<Enter>"}
+	go pressKey(uiEvents, input)
+
+	var entry menu.Entry = &DirOption{label: "root dir", path: "./testdata"}
+	var err error = nil
+	for {
+		if dirOption, ok := entry.(*DirOption); ok {
+			entry, err = dirOption.exec(uiEvents)
+			if err != nil {
+				t.Errorf("Fail to execute option (%q)'s exec(): %+v", entry.Label(), err)
+				break
+			}
+		} else if iso, ok := entry.(*ISO); ok {
+			if iso.label != wanted.label || iso.path != wanted.path {
+				t.Errorf("Get wrong chosen iso. get %+v, want %+v", iso, wanted)
+			}
+			break
+		} else {
+			t.Errorf("Unknow type, get a %T of entry %+v", entry, entry)
```

Comment:
Nit: `t.Errorf("Unknown type. got entry of type %T, wanted DirOption or ISO", entry)`